README.md

```diff
@@ -1 +1,65 @@
-# terraform-aws-named-subnets
\ No newline at end of file
+# terraform-aws-named-subnets
+
+Terraform module for named `subnets` provisioning.
+
+
+## Usage
+
+```terraform
+module "private_subnets" {
+  source             = "git::https://github.com/cloudposse/terraform-aws-named-subnets.git?ref=master"
+  name               = "${var.name}"
+  namespace          = "${var.namespace}"
+  stage              = "${var.stage}"
+  names              = ["charlie", "echo", "bravo"]
+  vpc_id             = "vpc-1234"
+  base_cidr          = "10.0.1.0/24"
+  availability_zones = ["us-east-1a", "us-east-1b"]
+  nat_enabled        = "true"
```

Comment:
I won't work properly for `private` subnets, because NAT GW should be in `public` subnet.

Reply:
NAT GW can't be created without EIP
https://www.terraform.io/docs/providers/aws/r/nat_gateway.html

Reply:
or, what you mean?

Reply:
@comeanother  is referring to the fact that nat gateways only work in public subnets, so the caller must be aware that they can only use the `nat_gateway` flag on the private subnet. it only works if there's an internnet gateway associated with the subnet.

Reply:
@osterman, thats out of date, module has been reworked